docs/process/release.md

```diff
@@ -208,3 +219,23 @@ If you don't have the firebase CLI installed, you can install it using
 `npm install --global firebase-tools` (or use your package manager of choice).
 
 This is detailed in [`etc/cli.angular.io/README.md`](https://github.com/angular/angular-cli/blob/master/etc/cli.angular.io/README.md).
+
+## Publishing a Major Version
+
+For the first release of a major version, say `v10.0.0`, checkout the patch branch
```

Comment:
> checkout the patch branch

Nit: I guess it is the `10.0.x` branch, but wouldn't we call this the major? Probably good to avoid name collisions with other branches.

Reply:
renamed patch branch to major branch